src/main/flight/feedforward.c

```diff
@@ -34,25 +34,34 @@
 #include "feedforward.h"
 
 static float setpointDelta[XYZ_AXIS_COUNT];
-
-typedef struct laggedMovingAverageCombined_s {
-     laggedMovingAverage_t filter;
-     float buf[4];
-} laggedMovingAverageCombined_t;
-
-laggedMovingAverageCombined_t  setpointDeltaAvg[XYZ_AXIS_COUNT];
-
 static float prevSetpoint[XYZ_AXIS_COUNT];
 static float prevSetpointSpeed[XYZ_AXIS_COUNT];
 static float prevAcceleration[XYZ_AXIS_COUNT];
 static float prevRcCommandDelta[XYZ_AXIS_COUNT];
-static uint8_t goodPacketCount[XYZ_AXIS_COUNT];
+static uint8_t duplicateCount[XYZ_AXIS_COUNT];
 static uint8_t averagingCount;
 static float feedforwardMaxRateLimit[XYZ_AXIS_COUNT];
 static float feedforwardMaxRate[XYZ_AXIS_COUNT];
 
+static float feedforwardTransitionFactor;
```

Comment:
There does not seem to be a reason to move these away from being const variables in function scope, as they are never changed and never used outside of `feedforwardInit()`.

Reply:
Thanks for clarifying that point, Mike.

Reply:
I've put them back and you were entirely correct, no change in flash size, works fine.